### DISCUSSION TITLE
How to install Firefox from github with Obtainium?

### DISCUSSION THREAD

**Discussion by DSDV** [ImranR98/Obtainium#1062] [Q&A] [answered]
Hi all (: 
i got to install several apps from git-hub with obtaiuium which worked great <3

Now i try to install Firefox ( https://github.com/mozilla-mobile/firefox-android/releases ) 

i got it to filter releases for "Firefox*"
and it shows "Fenix-v119.0.1" as latest version. which i think is correct. 

But the install button is still greyed out :( and i know that there are working apks cause i tested it with manual download. 

Any help is appreciated (:

**Comment by camilosampedro** [comment, marked as answer]
Do you have it already installed by any chance?

I had to uninstall it before importing it in Obtainium. The "Firefox*" filter in the release title was enough for me.

If you want to target beta releases, you can tick the "Include prereleases" option.

**Reply by DSDV** [reply]
to be hones it just reflashed my phone :/ 
But it might have been the problem that i manually installed it from GitHub before ( : 
Good find
